config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleDefinedEntities.java

```diff
@@ -14,7 +14,7 @@ public void setDefinedEntities(Collection<DefaultMetadata> definedEntities) {
         this.definedEntities = definedEntities;
     }
 
-    static class DefaultMetadata implements Metadata {
+    public static class DefaultMetadata implements Metadata {
```

Comment:
You can move this out to its own class. This would be useful later as well.

Reply:
done